x11-packages/shared-mime-info/postinst

```diff
@@ -1,14 +1,6 @@
 #!@TERMUX_PREFIX@/bin/bash
 set -e
-PREFIX="@TERMUX_PREFIX@"
 
-if [ "${1}" = "triggered" ]; then
-	if [ -x "${PREFIX}/bin/update-mime-database" ]; then
-		update-mime-database "${PREFIX}/share/mime"
-	fi
-	exit 0
-fi
+update-mime-database "@TERMUX_PREFIX@/share/mime"
+exit 0
```

Comment:
Is this really needed? It does not make sense as a last command of script.

Reply:
are you talking about exit 0?

Reply:
Yep.